test/run-test.sh

```diff
@@ -22,7 +19,15 @@ fi
 npx cdk --no-version-reporting -a ${cdk_app} synth > ${actual}
 
 if [ "${1:-}" == "update" ]; then
+  hash="$(cat ${actual} | md5sum | cut -c1-6 | xargs)"
+  export TEST_STACK_NAME="delivlib-test-${hash}"
   npx cdk --no-version-reporting -a ${cdk_app} deploy ${2:-} ${3:-} ${4:-}
+  echo "Stack deployed, now, go to the console and wait for the pipeline to fully stabalize"
+  echo "Press ENTER to confirm that pipeline is green"
```

Comment:
You're not telling me what I should do if it isn't 😱

Reply:
"Go figure" would probably be the recommendation.